core/src/main/java/org/infinispan/configuration/cache/BackupConfigurationBuilder.java

```diff
@@ -148,6 +148,9 @@ public void validate() {
          throw new CacheConfigurationException("It is required to specify a 'failurePolicyClass' when using a " +
                                                 "custom backup failure policy!");
       }
+      if (useTwoPhaseCommit && strategy == BackupConfiguration.BackupStrategy.ASYNC) {
+         throw new CacheConfigurationException("'two-phase-commit' can be used only with SYNC backup strategy!");
```

Comment:
IMO, it is necessary. Looking at the code (https://github.com/infinispan/infinispan/blob/master/core/src/main/java/org/infinispan/xsite/BackupSenderImpl.java#L248) it looks like it doesn't send anything to the remote site. 

I agree that the XSD should be updated and message should be in the ```Log``` interface.